.github/workflows/ecosystem-updates.yml

```diff
@@ -0,0 +1,24 @@
+name: Update Test Ecosystem Plugins
+
+on:
+    schedule:
+        - cron: "0 0 * * 1"
```

Comment:
Continuing @fisker's https://github.com/eslint/eslint/pull/19643#discussion_r2083532460: how frequently should this cron job run?

Reply:
My only concern with once a day is that it could cause a lot of PRs to be sent if the ecosystem projects are releasing a lot.

Reply:
Once a week?

I think fundamentally we just want to be sure it's run at least once in between releases to catch any issues before the next release.